other/pyramidofnumbers/pyramidofnumbers.go

```diff
@@ -0,0 +1,40 @@
+// Program in Golang to print Pyramid of Numbers
+/* author Sreejita Roy (https://github.com/pseudonerd16)*/
+
```

Comment:
I am not aware of any use of this particular algorithm. Please provide more information.